crowdsec-docs/unversioned/cti_api/taxonomy/cti_fields.md

```diff
@@ -188,8 +251,25 @@ The range to which the IP belongs.
 
 Evaluate the noisiness of an IP address, from a scale of 0 (not noisy) to 10 (extremely noisy)
 
+## `background_noise`
+
+> type: **string**
+
+```json
+"background_noise" : "malicious"
+```
+
+The background_noise level of the IP address.
```

Comment:
The level of background noise of an IP address is an indicator of its internet activity intensity.